lib/Transforms/LayoutOptimization/Patterns.cpp

```diff
@@ -55,15 +55,21 @@ LogicalResult HoistArgLayouts::matchAndRewrite(
         llvm::to_vector(maybeLayoutOps),
         [](auto& maybeLayoutOp) -> std::optional<tensor_ext::LayoutAttr> {
           if (!maybeLayoutOp.has_value()) return std::nullopt;
-          return maybeLayoutOp->getToLayout();
+          // Hoisting new layouts is unsupported.
```

Comment:
Remind me why? Isn't this pattern just moving the layout to the func arg as an attribute? If still valid, maybe slap a todo here